### DISCUSSION TITLE
Autosqaush fixup commits on rebase

### DISCUSSION THREAD

**Discussion by MarshallAsch** [community/community#37340] [Pull Requests]
I have recently started using fixup commits in PRs when making corrections that I do not actually want to have separate commits for but having as separate commits for the review process is nicer. 

Having an option to squash the fixup commits in `merge`, or `rebase` PR strategies would be nice so that it does not need to be done locally before merging.

**Comment by ossanna16** [comment]
Hi there @MarshallAsch and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
